client/vendors.ts

```diff
@@ -0,0 +1,2 @@
+import 'angular2/bundles/angular2-polyfills';
+import 'es6-shim';
```

Comment:
Maybe, it will be great to add some typings:
"globalDependencies": {
    "es6-shim": "registry:dt/es6-shim#0.31.2+20160317120654",
} 
and after that add script "postinstall": "typings install",